changelog/6484.misc.md

```diff
@@ -0,0 +1,19 @@
+## Summary
+- Renamed `templates` to `responses` across Rasa code base and documentation.
+- Renamed `response_templates` to `responses` across the Rasa code base.
+- Renamed `template_name` and `response_name` to `utter_action` in Rasa Core and rasa/shared.
+- Renamed `utter_templates` to `responses` in rasa/shared.
+
+### NLG Server
+- Changed request format to send `response` instead of `template` as a field.
+
+### rasa.core.agent
+- The terminology `template` is deprecated and replaced by `response`. Support for `template` from the NLG response will be removed in Rasa Open Source 3.0.0.
```

Comment:
let's add a link to the docs for NLG server

Reply:
`https://rasa.com/docs/rasa/nlg` or there is a way to generate a link that is environment agnostic?

Reply:
yes, see e.g. this as example https://github.com/RasaHQ/rasa/blob/f8e0000f39fc0da6f74bef55c1d3007a6a5984b7/changelog/7848.improvement.md#L1